package.json

```diff
@@ -28,7 +28,8 @@
 		"test:build": "tsx tools/checkBuild.ts",
 		"test:tables": "npx markdown-table-formatter \"docs/**/*.{md,mdx}\" --check",
 		"fix:tables": "npx markdown-table-formatter \"docs/**/*.{md,mdx}\"",
-		"decorate:docs": "node tools/doxygen/generate-social-sdk-mapping.js && node tools/doxygen/decorate-social-sdk-references.js"
+		"decorate:docs": "node tools/doxygen/generate-social-sdk-mapping.js && node tools/doxygen/decorate-social-sdk-references.js",
+		"prepare": "npx husky || true"
```

Comment:
Note: This is needed so the CI doesn't fail (since dev dep) ref: https://typicode.github.io/husky/how-to.html#ci-server-and-docker

Reply:
This documentation is slightly confusing and this doesn't apply here, we're installing all deps. I would also highly suggest `is-ci || husky` rather than this since it will automatically skip in all ci runs (can remove the env settings)

Reply:
Thanks! We are talking about https://www.npmjs.com/package/is-ci right?